ui/src/css/_common.scss

```diff
@@ -0,0 +1,80 @@
+.mdc-button {
+  text-transform: none;
+}
+
+.mdc-card__primary-action {
+  padding: 0 8px;
+}
+
+.mdc-card__actions {
+  padding: 0;
+}
+
+.mdc-tab-indicator .mdc-tab-indicator__content--underline {
+  background-color: #1a73e7;
+}
+
+.mdc-typography--headline5 {
+  margin: 0.5em 0;
+}
+
+.mdc-text-field .mdc-text-field__input {
+  caret-color: #757575;
+}
+
+body {
+  margin: 0;
+}
+
+input[type=file] {
+  display: none;
+}
+
+@media only screen and (min-width: 482px) {
+  /**
+   * We still want a largely vertical/linear view on most pages even on wider
+   * screens, but we put a border around them so they don't look weird.
+   */
+  .pf-linearwrap {
+    border-left: 1px solid #dadce0;
+    border-right: 1px solid #dadce0;
+    margin: 0 auto;
+    max-width: 480px;
+  }
+}
+
+.pf-button-primary {
+  @include mdc-button-filled-accessible(#1a73e7);
+  @include mdc-button-ink-color(#fff);
+}
+
+.pf-button-secondary {
+  @include mdc-button-filled-accessible(#fff);
+  @include mdc-button-ink-color(#1a73e7);
+}
+
+/*
+ * These classes are to support the section headers that are styled like this:
+ *      --- Title or whatever ---
+ */
+.endbars-headerline {
+  color: #80868b;
+  font-size: 18px;
+  font-weight: bold;
+  padding: 0 5px;
+}
+.endbars-headerline:before, .endbars-headerline:after {
+  background-color: #dadce0;
+  content: "";
+  display: inline-block;
+  height: 4px;
+  position: relative;
+  vertical-align: middle;
+  width: 32px;
+}
+.endbars-headerline:before {
+  right: 8px;
```

Comment:
Can be tested later, but does this work under RTL?

May be replaced with inset-inline-end which automatically becomes right or left based on LTR/RTL:
https://developer.mozilla.org/en-US/docs/Web/CSS/inset-inline-end
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties
But it's a relatively new CSS feature, so I'm not sure if it supports all browsers we want to support:
https://caniuse.com/#feat=css-logical-props

Reply:
It seems to work (I set `dir="rtl"` on the body element and got the expected result at least). Here's a screenshot:
![pf_endbars_rtl](https://user-images.githubusercontent.com/9059304/51352095-99317b80-1a7a-11e9-97f5-926588845b30.png)

Reply:
Hm that's interesting. I believe you attached `dir="ltr"` to an endbars-headerline element in the code. Does it work even if you remove it?

Reply:
Ha, I'd forgotten about that; that explains why it works. No, it doesn't work without that.
Since the text in these is always centered, I'm not sure we really need to worry about the text direction not matching the language (although it might be a problem if there's punctuation involved?).
I haven't implemented anything to actually set the text direction on the root element yet; I'll add a todo here to make sure this works as intended when I do the rest of what's needed for RTL. If there's a problem I think we might be able to use dir="auto" on the inner span to resolve it, but I'm not sure.

Reply:
Sounds good to test it later. I don't really know what is affected by the "dir" attribute. e.g., It would be definitely bad if it renders Arabic text LTR if `dir="ltr"` is specified here.